src/Uno.Extensions.Templates/content/unoapp-extensions/azure-pipelines.yml

```diff
@@ -0,0 +1,220 @@
+            target_platform: 'android'
+
+        - template: /azure/steps/android.yml
+          parameters:
+            package_id: 'Android'
+            project_to_build: 'MyExtensionsApp.Mobile/MyExtensionsApp.Mobile.csproj'
+      pool:
+        vmImage: macOS-latest
+
+    ###################################################
+    #                    IMPORTANT                    #
+    # Store certificates and mobile provision files   #
+    # in secure files, then just replace the jks      #
+    # variable down below                             #
+    # with the appropiate filename                    #
+    ###################################################
+
+    - job: Android_Signed   
+      timeoutInMinutes: '${{ variables.jobTimeout }}'
+      steps:
+        - template: /azure/steps/install-dependencies.yml  
+          parameters:
+            target_platform: 'android'
+        - template: /azure/steps/android.yml
+          parameters:
+            sign_package: 'true'
+            certificate:  '' #YOUR_CERT.jks stored in secureFiles
+            certificate_password: 'contoso'
+            certificate_alias: 'contoso'
+            package_id: 'signed_android'
+            project_to_build: 'MyExtensionsApp.Mobile/MyExtensionsApp.Mobile.csproj'
+      pool:
+        vmImage: macOS-latest         
+#endif
+
+#if (ios) 
+    - job: iOS
+      timeoutInMinutes: '${{ variables.jobTimeout }}'
+      steps:
+        - template: /azure/steps/install-dependencies.yml  
+          parameters:
+            target_platform: 'ios'
+
+        - template: /azure/steps/mobile.yml
+          parameters:
+            project_to_build: 'MyExtensionsApp.Mobile/MyExtensionsApp.Mobile.csproj'
+            target_framework: 'net$ciTargetDotNetVersion$.0-ios'
+      pool:
+        vmImage: windows-latest
+
+    ###################################################
+    #                    IMPORTANT                    #
+    # Store certificates and mobile provision files   #
+    # in secure files, then just replace the          #
+    # p12 and mobileprovision variables down below    #
+    # with the appropiate filenames                   #
+    ###################################################
+
+    - job: iOS_Signed
+      timeoutInMinutes: '${{ variables.jobTimeout }}'
+      steps:
+        - template: /azure/steps/install-dependencies.yml  
+          parameters:
+            target_platform: 'ios'
+        - template: /azure/steps/ios.yml
+          parameters:
+            package_id: 'signed_ios'
+            project_to_build: 'MyExtensionsApp.Mobile/MyExtensionsApp.Mobile.csproj'
+            certificate: '' #YOUR_CERT.p12 stored in secureFiles
+            certificate_password: 'contoso'
+            provisioning_profile: ''  #YOUR_PROV_PROFILE.mobileprovision stored in secureFiles
+            codesign_provision: 'MyExtensionsApp'
+            codesign_key: 'Apple Development: Contoso Contoso (572224A42L)'
+      pool:
```

Comment:
See previous comment on android signing job about signing parameters - we should only enable this job if those parameters are available and should show warning if they're not